app/src/main/java/mozilla/lockbox/presenter/ItemDetailPresenter.kt

```diff
@@ -117,8 +128,8 @@ class ItemDetailPresenter(
 
     private fun handleClicks(clicks: Observable<Unit>, withServerPassword: (ServerPassword) -> Unit) {
         clicks.subscribe {
-                this.credentials?.let { password -> withServerPassword(password) }
-            }
+            this.credentials?.let { password -> withServerPassword(password) }
```

Comment:
Did an auto-formatter get a little aggressive? (-:

I think the original spacing should be preserved, to help reinforce this is in the middle part of a chain.